### DISCUSSION TITLE
https://angular-love-client.pages.dev/testowanie-jednostkowe-widokow-z-blokami-defer

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#275] [General]
# https://angular-love-client.pages.dev/testowanie-jednostkowe-widokow-z-blokami-defer

Angular.love - ciekawostki oraz rozwiązania dla średnio-zaawansowanych oraz zaawansowanych developerów Angulara.

https://angular-love-client.pages.dev/testowanie-jednostkowe-widokow-z-blokami-defer

**Comment by OllieS2137** [comment]
Test
